walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/Utils.java

```diff
@@ -0,0 +1,15 @@
+package com.google.sps;
+
+import java.util.Set;
+
+public class Utils {
+    public static boolean hasIntersection(Set<String> eventAttendees, Set<String> requestAttendees) {
```

Comment:
public method must have javadoc

Reply:
You can use `Collections.disjoint`

Reply:
Fixed by moving it back to private member method